src/main/kotlin/com/learnspigot/bot/entity/UserProfile.kt

```diff
@@ -26,11 +27,15 @@ data class UserProfile(
         reputation.add(point)
         acknowledgeRep(point, leaderboardManager, guild)
     }
-
+    fun changeRepuationToPing() {
```

Comment:
This is essentially a redundant setter. We can just do `profile.reputationPing = !profile.reputationPing` wherever this code is called.